src/Redux/index.js

```diff
@@ -0,0 +1,18 @@
+/*eslint-disable*/
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from './App';
+import store from './redux/configureStore';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Provider>
+  </React.StrictMode>,
+);
```

Comment:
- [ ] From the file structure shared with you this file should be in the `src` directory, not `src/Redux`. Kindly fix that.